### PR TITLE
KupreichikA hw9 task2

### PR DESCRIPTION
**Pull request by akupreychik**

### CODE REVIEW

File: homework/Kupreichik Anton/HomeWok_9/ex2_UniqueWordCounter/com.company/service/UniqueWordCounter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        addWord("Приветик");
+
+        System.out.println("Максимальное кол-тво добавлений:" + getMostFrequentWord() + " раз(a)" );
+        printWordsFrequency();
+        words.keySet().stream().forEach(System.out::println);
+    }
+
+    Map<String, Integer> words = new HashMap<>();
+
+    public void addWord(String word) {
```
Comment (javaGuruBY) [inline homework/Kupreichik Anton/HomeWok_9/ex2_UniqueWordCounter/com.company/service/UniqueWordCounter.java:28]: good

**Review by javaGuruBY — APPROVED** [review]
lgfm

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

**Review by vladis1350 — APPROVED** [review]
Lgfm

**Pull request merged**